src/constructs/core/stack.test.ts

```diff
@@ -41,6 +43,25 @@ describe("The GuStack construct", () => {
     expect(stack).toHaveGuTaggedResource("AWS::IAM::Role");
   });
 
+  it("should prefer context values for repository information", () => {
+    const stack = new GuStack(new App({ context: { [ContextKeys.REPOSITORY_URL]: "my-repository" } }), "Test", {
```

Comment:
This is atypical use of CDK context, as one would usually create a `cdk.json` file instead. We cannot do this here as we're building a library, rather than an application stack.